frame/support/procedural/src/storage/print_pallet_upgrade.rs

```diff
@@ -323,10 +323,6 @@ pub mod pallet {{
 	#[pallet::generate_store({store_vis} trait Store)]
 	pub struct Pallet{decl_gen}(PhantomData{use_gen_tuple});
 
-	/// Old name for pallet.
```

Comment:
Why didn't you used `quote!` for all this code?

Reply:
If I remember correctly I tried with `quote!` but then indentation was weird and there was spaces between some tokens. Though I didn't tried much.